release/linux-specific/omegat.desktop

```diff
@@ -0,0 +1,24 @@
+[Desktop Entry]
```

Comment:
Does this file not need any special handling in e.g. build.gradle?

Reply:
It is a file to copy when linux-install.sh executed. That's jut it.

Reply:
There is an idea that we put a template here and generate translations from Bundle.properties.
This will help translator team to handle desktop entry translation.

Reply:
It is not obvious merit for my previous idea.